docs/src/_reference/command-line-interface.md

```diff
@@ -732,11 +731,69 @@ meltano job remove <job_name>
 
 ### Examples
 
+##### Tasks and Blocks
+
+A task in a job consists of one or more _blocks_. A block is simply a set of related plugins that will be executed serially.
+For example, you can create a job that consists of a single tap and target:
+
+```bash
+meltano job add tap-gitlab-to-target-postgres --tasks "tap-gitlab target-postgres"
+```
+
+This would add the following to your `meltano.yml`:
+
+```yaml
+jobs:
+  - name: tap-gitlab-to-target-postgres
+    tasks:
+      - tap-gitlab target-postgres
+```
+
+Plugins in the same block should be separated by a space and blocks should be separted by a comma.
```

Comment:
```suggestion
Plugins in the same block should be separated by a space and blocks should be separated by a comma.
```

Reply:
Above it says "Add a new job with multiple tasks by passing arrays in yaml format, where each item representing a run command.", so it seems that the comma/list format separates _tasks_, not _blocks_ as suggested here. Could that be part of why I'm confused?

"where each item representing a run command" also suggests that `--tasks "[foo bar, baz qux]"` is equivalent to `meltano run foo bar` followed by `meltano run baz qux`. It's still not clear to me why that is preferable to `--tasks "foo bar baz qux"` and `meltano run foo bar baz qux`, though.

Reply:
> so it seems that the comma/list format separates tasks, not blocks as suggested here. Could that be part of why I'm confused?

Correct, blocks are not comma deliminated you run `meltano run tap target` not `meltano run tap, target` .

> It's still not clear to me why that is preferable to --tasks "foo bar baz qux" and meltano run foo bar baz qux, though.

Because in airflow this yields exactly that, two tasks:

1. `meltano run foo bar`
2. `meltano run baz qux`